### ISSUE TITLE
Draw.IO glitching when using iPad's pro magic keyboard with trackpad

### ISSUE THREAD

**Issue by sokzkota** [jgraph/drawio#2758]
It's not a big issue, but it seems when using trackpad to move elements, iPad tries to mark  the diagrams as text causing some really ugly blue squares everywhere. Please see the screen. 

Problem does not exist when using touch 

<img width="1366" alt="image" src="https://user-images.githubusercontent.com/51513615/163560846-6c374b5a-3f9b-4586-9873-bd44cfd90253.png">

**Comment by DreamShark-Bytes** [comment]
This issue is occurring with me too. 
Tested Safari and Chrome, and appears to only affect Safari. 

I’m glad this doesn’t break the program, but it makes things more difficult.

**Comment by DreamShark-Bytes** [comment]
Never mind, it’s happening on Chrome too now. 
Man, this is disorientating. 

the blue squares are PULSATING (very frequently flickering in and out of view) 

I think this could give someone a seizure if they have photosensitive epilepsy. 

[here’s a clip of the visual glitch in action](https://drive.google.com/file/d/1FWTQuJUD4Koa_qiqvAjzm_pR8Eq5a7HY/view?usp=drivesdk)

**Comment by DreamShark-Bytes** [comment]
I switched to using the keyboard and trackpad b/c the UX of the website just isn’t user-friendly enough for that. 
I’m not saying it has to be be touch-screen friendly, but it’s certainly equally as unwieldy using the only alternative I’ve tried because of this bug.

**Comment by DreamShark-Bytes** [comment]
Is this issue not getting traction because it’s missing information from the bug tracking template?
I’m noticing a lot of missing fields such as:

- Preflight Checklist
- Describe the bug
- To reproduce
- Expected behavior
- Screenshots (included but not marked)
- Draw.io version
- smart phone info
- additional context

Would the collaborators need this information? I can put it in a comment? Or perhaps a resubmission of this bug?
Let me know. 

Attempting to use the iPad without the trackpad is proving very difficult.

**Comment by alderg** [comment]
What keyboard and trackpad are you using? What are the difficulties on the iPad without the trackpad? (If it's rubberband selection, try touch and hold.)

**Comment by DreamShark-Bytes** [comment]
The issue appears to affect  more than just trackpads, but any mouse/pointer device (except the apple pencil).
Here are the devices I’ve encountered the issue:
1.  the trackpad on the apple magic keyboard
2. standalone apple magic trackpad
3. razer viper mouse

The issue does not occur when using touch input or the apple pencil. Nor does it appear to occur on a mac laptop.

**Comment by davidjgraph** [comment]
I can see it on safari on ipad pro with a trackpad on the keyboard. It's very minor and doesn't happen in chrome. Personally, it's barely noticeable and there's a simple workaround, use the screen.

**Comment by DreamShark-Bytes** [comment]
Just tested again in Chrome and it still occurs. Tested the Magic Keyboard (case) and Razer Viper mouse.

I'm actually seeing something new,  for the iPad Pro. Clicking and dragging appears to sporadically highlight. 
Here is what I'm seeing:
- about 40% of the time: it doesn't show the "rubberband tool" until I stop clicking. I have to click again to remove the highlight box and when it disappears, it doesn't select any items.
- about 50% of the time: the same flashes showcased in the video I sent. 
- and the remaining 10% of the time it appears to work correctly. 

Doesn't seem consistent. Regardless, not really usable for my needs. Getting a headache from the flashing, regardless of it being slightly reduced in Chrome, is not viable for using the input methods of a mouse/trackpad. 

Touch screen is still not ideal. Using apple pencil kind of works. Fingers are awkwardly frustrating. 
Two-finger tap for undo would be wonderful. Thinking about gestures used by drawing programs, like Procreate. 

Even if there is a "work-around", is still issue good to remain open? 
I would REALLY like this issue to be fixed. I've gone through many programs trying to get a diagram program to work on a project between my computer at home, and iPad (syncing to the cloud) and this was the closest one to idea. Only thing holding it back for me as a user is this trackpad/mouse thing.

**Comment by DreamShark-Bytes** [comment]
I need to rephrase
This is a BUG, why is this not labeled as a bug yet? 

Here is a video of it happening on Chrome (version 105.0.5195.129)

- First click did nothing until I released my mouse click then it was a selection box until I clicked again and it finally selected items. 
- Second attempt was same type of click, but instead is created the flashing. 

![Automatic](https://user-images.githubusercontent.com/70111379/190870813-1c28cebc-afe1-473a-9f78-551a373b2d11.gif)

Here is Safari

![Automatic](https://user-images.githubusercontent.com/70111379/190870820-27133685-ffd1-4bf9-8c51-c7688b7f9e7f.gif)
